.github/workflows/test_cocoapods_integration.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
 
   pods:
-    runs-on: macos-12-xl
+    runs-on: macos-14-xlarge
```

Comment:
Is this update still needed? (I agree that it makes sense to upgrade but was wondering if it's required - did you run the compatibility tests already?)

Reply:
Strictly - I think it is not needed anymore, as we have downgraded the code. 
But if you compare with the other actions, it has the same configuration we are using in `test-SPM-integration.yml`

Compatibility tests - passed yesterday. 

let me know if you need to change/revert this. Happy to.